src/planscape/forsys/get_forsys_inputs_test.py

```diff
@@ -154,10 +162,55 @@ def test_raises_error_for_url_params_project_areas_missing_id_field(self) -> Non
         self.assertEquals(
             str(context.exception), 'project area missing field, "id"')
 
-    def test_reads_from_db(self) -> None:
+
+class TestForsysProjectAreaRankingRequestParams_ReadFromDb(TestCase):
+    def setUp(self):
+        self.base_condition = BaseCondition.objects.create(
+            condition_name="name", condition_level=ConditionLevel.ELEMENT)
+        self.condition1 = Condition.objects.create(
+            condition_dataset=self.base_condition, raster_name="name1")
+        self.condition2 = Condition.objects.create(
+            condition_dataset=self.base_condition, raster_name="name2")
+
+        self.user = User.objects.create(username='testuser')
+        self.user.set_password('12345')
+        self.user.save()
+
+        self.geometry = {'type': 'MultiPolygon',
+                         'coordinates': [[[[1, 2], [2, 3], [3, 4], [1, 2]]]]}
+        stored_geometry = GEOSGeometry(json.dumps(self.geometry))
```

Comment:
to fix the failing test: replace with "self.stored_geometry ="

Reply:
thanks!